labs/bbb-webrtc-sfu/lib/screenshare/ScreenshareProcess.js

```diff
@@ -4,7 +4,7 @@ const ScreenshareManager = require('./ScreenshareManager');
 const BaseProcess = require('../base/BaseProcess');
 const C = require('../bbb/messages/Constants');
 
-const manager = new ScreenshareManager(C.TO_SCREENSHARE, [C.FROM_BBB_TRANSCODE_SYSTEM_CHAN], C.SCREENSHARE_MANAGER_PREFIX);
+const manager = new ScreenshareManager(C.TO_SCREENSHARE, [C.FROM_BBB_TRANSCODE_SYSTEM_CHAN, C.FROM_AKKA_APPS], C.SCREENSHARE_MANAGER_PREFIX);
```

Comment:
I didn't know we could have (a group of) optional params in the middle of the params list. Good to know! :+1:

Reply:
We actually shouldn't have, optional parameters should come last :man_facepalming:. I'll change that later, mea culpa.